definitions/json/ComplexTypes.json

```diff
@@ -21,6 +21,93 @@
     "ElementLabel": "Short Description",
     "SecurityClassification": "Public"
   },
+  {
+    "ID": "AdrDocumentUploadDetails",
+    "ListElementCode": "typeOfDocument",
+    "FieldType": "FixedList",
+    "ElementLabel": "Type of Document",
+    "FieldTypeParameter": "fl_DocumentType",
+    "SecurityClassification": "Public"
+  },
+  {
+    "ID": "AdrDocumentUploadDetails",
+    "ListElementCode": "uploadedDocument",
+    "FieldType": "Document",
+    "ElementLabel": "Document Link",
+    "SecurityClassification": "Public"
+  },
+  {
+    "ID": "AdrDocumentUploadDetails",
+    "ListElementCode": "shortDescription",
+    "FieldType": "Text",
+    "ElementLabel": "Short Description",
+    "SecurityClassification": "Public"
+  },
+  {
+    "ID": "AdrDocumentUploadDetails",
+    "ListElementCode": "creationDate",
+    "FieldType": "Date",
+    "ElementLabel": "Date Uploaded",
+    "SecurityClassification": "Public"
+  },
+  {
+    "ID": "PiiDocumentUploadDetails",
+    "ListElementCode": "typeOfDocument",
```

Comment:
only need document, short desc, and date. can remove type of document

Reply:
removed as suggested.